rotkehlchen/tests/unit/decoders/test_balancer.py

```diff
@@ -230,7 +230,7 @@ def test_balancer_v1_join(database, ethereum_manager, eth_transactions):
             ethereum_inquirer=ethereum_manager.node_inquirer,
             transactions=eth_transactions,
         )
-    events = decoder.decode_transaction(transaction=transaction, tx_receipt=receipt)
+    events, _ = decoder.decode_transaction(transaction=transaction, tx_receipt=receipt)
```

Comment:
I fear that it will happen as with the decoding output that we end up adding return arguments and having to refactor all the code. Why not to create a namedtuple and isntead of doing this around the code just

```suggestion
   events = decoder.decode_transaction(transaction=transaction, tx_receipt=receipt).events
```

Reply:
It would make sense, but I think not right now, since atm it's only 2 returned values. For when it grows -- yes, sure, will make sense to put it in a separate class.